### ISSUE TITLE
Seems like optimizer.step() has been overridden after learning rate scheduler initialization

### ISSUE THREAD

**Issue by AlphaPlusTT** [JDAI-CV/fast-reid#496]
Hi, I'm using your run_lm_finetuning.py script. It works but I would like to know why in the starting of the training, I get the following warning: 

```
/home/111/anaconda3/envs/fastreid/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:114: UserWarning: Seems like `optimizer.step()` has been overridden after learning rate scheduler initialization. Please, make sure to call `optimizer.step()` before `lr_scheduler.step()`. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
```

**Comment by L1aoXingyu** [comment]
Updated, this warning is gone.
